index.html

```diff
@@ -52,7 +52,73 @@ <h3 class="stories">Multi-Post Stories</h3>
     <a href="#"><li class="tag3">Bootstrap</li></a>
     <a href="#"><li class="tag4">Ruby</li></a>
   </ul>
-  <div><a href="#" class="see__project">See Project</a></div>
+    <a href="#" class="see__project">See Project</a>
+    <section class="section3">
+      <h4 class="section3-heading">Proffesional Art Printing Data</h4>
+      <p class="section3-paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+      <ul class="section3-items">
+        <a href="#" class="section3-item1"><li>HTML</li></a>
+        <a href="#" class="section3-item2"><li>Bootstrap</li></a>
+        <a href="#" class="section3-item3"><li>Ruby</li></a>
+      </ul>
+      <a href="#" class="section3__project">See Project</a>
+    </section>
+    <section class="section3">
+      <h4 class="section3-heading">Proffesional Art Printing Data</h4>
+      <p class="section3-paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+      <ul class="section3-items">
+        <a href="#" class="section3-item1"><li>HTML</li></a>
+        <a href="#" class="section3-item2"><li>Bootstrap</li></a>
+        <a href="#" class="section3-item3"><li>Ruby</li></a>
```

Comment:
- [ ] Your technologies should have the `a` tag inside the `li` tag, not the other way around since the children of the `ul` tag are the `li` tags. 
- [ ] Also your styling of the technologies still needs a little bit of work before they are good to go. Below are is your design and the design from the Figma template.
- [ ] This should also be applied to the rest of the project cards.
-  Your design:
![f4](https://user-images.githubusercontent.com/85113567/148490865-0eb12d24-0d49-4a53-bee6-068afe50943f.jpg)
- Figma's design:
![f3](https://user-images.githubusercontent.com/85113567/148490881-1214d2ac-4642-432e-9b44-d0ae56bed2b8.jpg)